flloat/ltlf.py

```diff
@@ -535,6 +605,15 @@ def _delta(self, i: PropositionalInterpretation, epsilon=False):
         """Apply the delta function."""
         return self.to_nnf()._delta(i, epsilon=epsilon)
 
+    def to_ldlf(self):
+        """Convert the formula to LDLf."""
+        return LDLfNot(
+            LDLfDiamond(
+                RegExpStar(RegExpPropositional(PLTrue())),
+                LDLfAnd([LDLfNot(self.f.to_ldlf()), LDLfNot(LDLfEnd())]),
```

Comment:
this looks wrong. It should be the dual of the Diamond formula of the eventually

Reply:
true, but making it a LTLfBox does not work..

Reply:
namely:
```
!F(!p) = G(p)

!<true*>(!p & !end) = [true*](p | end)

```

Reply:
> this looks wrong. It should be the dual of the Diamond formula of the eventually

Sorry I haven't seen the outer not. It is correct, however we could use the Box so the formula is simpler.